### ISSUE TITLE
How to get position of current order? Got undefined via getPosition()

### ISSUE THREAD

**Issue by vitalizis** [Reiryoku-Technologies/Mida#138]
I use ctrader and try to get position for current order but I got undefined. More details in comment

```
    const orders = await myAccount.getOrders('EURUSD');
    const id = 123; // exist order with this number in ctrader and open position with this number of order
    const order = orders.filter((item)=>item.id==id)[0]
    console.log(order); // CTraderOrder {} 
    const position = await order.getPosition()
    console.log(position); // undefined
```

**Comment by Vasile-Peste** [comment]
Hello, there are no historical positions. Positions are virtual, they represent the trades/assets that you hold.
So `order.getPosition()` returns only the open position. When a position is closed, you can't get it because it's not saved in any database. You have to get its trades with `getTrades()`
I hope this is clear

Thank you,
Vasile

**Comment by olehmalkov** [comment]
Hi, Vasile, Thank u!

**Comment by Vasile-Peste** [comment]
The fix has been delivered with [Mida 2023.10.1](https://github.com/Reiryoku-Technologies/Mida/releases/tag/v2023.10.1), the bug affected only historical orders, I'm going to close this, please open a new issue if you have any other problem

Thank you!
